src/dune_tui/drawing/user_message_to_image.mli

```diff
@@ -0,0 +1,4 @@
+open Stdune
+
+(** [User_message_to_image.pp ~attr pp] converts a pretty-printer [pp] to a [Notty.image]. *)
+val pp : User_message.Style.t Pp.t -> Notty.image
```

Comment:
I question the organization of this `drawing` subdirectory. It's basically one module per function, why?

You could just keep a single drawing module and keep all the code there.

Reply:
Done.